runtime/src/genesis_utils.rs

```diff
@@ -200,6 +201,20 @@ pub fn activate_all_features(genesis_config: &mut GenesisConfig) {
     }
 }
 
+pub fn deactivate_features(genesis_config: &mut GenesisConfig, features_to_skip: &Vec<Pubkey>) {
+    // Remove all features in `features_to_skip` from genesis
+    for deactivate_feature_pk in features_to_skip {
+        if FEATURE_NAMES.contains_key(deactivate_feature_pk) {
+            genesis_config.accounts.remove(deactivate_feature_pk);
```

Comment:
Thinking out loud, the workflow we're using is to activate all features and then pull out ones we don't want. However, `genesis_config.accounts` is a `BTreeMap` so even if none of the features have been activated and are NOT present in the `BTreeMap`, `remove()` here will just gracefully do nothing.

Reply:
yessss correct. `remove()` will return None if the key is not in the map. but we don't do anything with the return value here. so nothing will happen